1.0/profile.ttl

```diff
@@ -0,0 +1,63 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX prof: <http://www.w3.org/ns/dx/prof/>
+PREFIX role: <http://www.w3.org/ns/dx/prof/role/>
+PREFIX sdo: <https://schema.org/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
+
+
+<http://www.opengis.net/def/geosparql>
+    a prof:Profile ;
+    dcterms:identifier "geosparql" ;
+    dcterms:created "2009-10-27"^^xsd:date ;
+    dcterms:modified "2012-07-19"^^xsd:date ;
+    dcterms:issued "2020-xx-xx"^^xsd:date ;
+    dcterms:publisher <https://www.ogc.org> ;
+    dcterms:title "GeoSPARQL Specification"@en ;
+    dcterms:description "This profile supports representing and querying geospatial data on the Semantic Web. GeoSPARQL defines a vocabulary for representing geospatial data in RDF, and it defines an extension to the SPARQL query language for processing geospatial data. It is presented in a series of artefacts: the vocabulary as an RDF ontology and the specification as a PDF document." ;
+    prof:hasResource 
+        <http://www.opengis.net/doc/IS/geosparql/1.0> ,
+        <http://www.opengis.net/ont/geosparql> ,
+        <http://www.opengis.net/def/geosparql/funcsrules> ,
+        <http://www.opengis.net/ont/sf> ;
+.
+
+<https://www.ogc.org>
+    a owl:NamedIndividual ;
+    a sdo:Organization ;
+    sdo:name "Open Geospatial Consortium" ;
+    sdo:url <https://www.ogc.org> ;
```

Comment:
It does two things: it establishes that the OGC is an Agent (or the derived `sdo:Organization`) and we just happen to use the same URI they use for their public website as our identifier for them, which is reasonable since we have no other sensible identifier, but also I'm hoping the OGC and others *will* publish stable Linked Data identifiers for themselves eventually. 

So, ideally we would use a better PID for the OGC and then link that to their website via `sdo:url`. Any suggestions for their PID? I have considered using a ROR, in this case <https://ror.org/00nz0j151> (see https://ror.org/search?query=open+geospatial+consortium).

Reply:
How about http://www.opengis.net/def/ogc? Wouldn't it stand to reason to have the OGC Naming Authority define the OGC?

Reply:
https://www.wikidata.org/entity/Q776903 
https://ror.org/00fsdxs93 is a better ROR IRI

Reply:
I've added Simon's suggested ROR URI for the OGC in Commit https://github.com/opengeospatial/ogc-geosparql/pull/58/commits/12ef349a1b6bbb84d5b7f1f2d7fd5193053c8728

I think this will have to do for now (internal SWG review). We can put it to the Naming authority for them to correct. They will be the ones who decide on the question _Wouldn't it stand to reason to have the OGC Naming Authority define the OGC?_